QuasarHelios2015/src/org/team1540/quasarhelios/ControlInterface.java

```diff
@@ -85,8 +77,23 @@ private static void setupRollers() {
 
         FloatInput cutoffAuto = mainTuning.getFloat("Trigger Threshold +M", 0.5f);
 
-        BooleanMixing.pumpWhen(QuasarHelios.manualControl, FloatMixing.floatIsAtLeast(Igneous.joystick2.getAxisSource(3), cutoffAuto), QuasarHelios.autoEjector);
-        BooleanMixing.pumpWhen(QuasarHelios.manualControl, FloatMixing.floatIsAtLeast(Igneous.joystick2.getAxisSource(4), cutoffAuto), QuasarHelios.autoLoader);
+        FloatInput leftTrigger = Igneous.joystick2.getAxisSource(3);
+        FloatInput rightTrigger = Igneous.joystick2.getAxisSource(4);
+
+        BooleanInputPoll rollersMode = Igneous.joystick2.getButtonChannel(5);
```

Comment:
Wouldn't this require holding a trigger while using the triggers? Does that work physically?